ipaserver/install/gcinstance.py

```diff
@@ -62,6 +63,21 @@
 
 ALL_SCHEMA_FILES = GC_SCHEMA_FILES
 
+# MS-DRSR 2.2.3 and 2.2.4, SPNs for a target DCs in AD DS
+# List of (primary principal, service alias)
+DC_ALIASES = (
+    # LDAP SPNs for client communication
+    ('ldap/$FQDN', 'ldap/$NETBIOS'),
+    ('ldap/$FQDN', 'ldap/$FQDN/$WORKGROUP'),
+    ('ldap/$FQDN', 'ldap/$FQDN/$DOMAIN'),
+    ('ldap/$FQDN', 'ldap/$NETBIOS/$WORKGROUP'),
+    ('ldap/$FQDN', 'GC/$FQDN/$DOMAIN'),
+    # DRS interface GUID SPN
+    ('ldap/$FQDN',
+     'E3514235-4B06-11D1-AB04-00C04FC2DCD2/$DOMAINGUID/$DOMAIN'),
+    # Basic SPN for RPC communications
+    ('cifs/$FQDN', 'GC/$FQDN/$DOMAIN'))
```

Comment:
We can't add the same alias GC/$FQDN/$DOMAIN for cifs/$FQDN and ldap/$FQDN as it will conflict with the uniqueness plugin configured on krbPrincipalName.

Reply:
It should be RPC, not GC -- a typo is here!